CMakeLists.txt

```diff
@@ -94,3 +98,23 @@ if(QHOTKEY_INSTALL)
 
     export(TARGETS qhotkey FILE QHotkeyConfig.cmake)
 endif()
+
+if(UNIX)
```

Comment:
Could you remove `if(UNIX)`? This settings won't hurt on other OSes. Even on Windows you can cross-compile Deb.